config/jobs/kubernetes/kops/build_grid.py

```diff
@@ -206,7 +217,13 @@ def build_test(cloud='aws',
     if should_skip_newer_k8s(k8s_version, kops_version):
         return None
 
-    kops_image, kops_ssh_user = distro_info(distro)
+    kops_image = distro_images[distro]
+    if kops_image is None:
+        raise Exception(f"missing required distro image for {distro}")
```

Comment:
This code will never be reached because `distro_images[distro]` will raise a KeyError if the key doesn't exist.  I think the generic KeyError is fine and we dont need to make error message any friendlier.

Same with the ssh user exception below.

Reply:
My python is a bit rusty...